manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper

```

```suggestion
PackageIdentifier: RobloxCorporation.Roblox
```
No need to call out the fact it's a boostrapper. The folder and file names will need to be updated with this change.

---

apply change